Java: 17052 Second try: do not expose error message (#17075)
github/codeql: java/ql/src/change-notes/2024-07-25-java-error-message-exposure.md

---
category: minorAnalysis
---

> * Now alerts about exposing `exception.getMessage()` in servlet responses are split out of `java/stack-trace-exposure` into its own alert `java/error-message-exposure` because this is a better fit. 

Reviewer: Minor rewording:
```suggestion
* Alerts about exposing `exception.getMessage()` in servlet responses are now split out of `java/stack-trace-exposure` into its own query `java/error-message-exposure`.
```
